api/src/main/java/org/asynchttpclient/request/body/generator/IFeedableBodyGenerator.java

```diff
@@ -0,0 +1,11 @@
+package org.asynchttpclient.request.body.generator;
+
+public interface IFeedableBodyGenerator {
```

Comment:
For consistency, I'd rather remove the `I` prefix and prefix the default implementation with `Default` or `Simple`.